src/test/java/SignUp.java

```diff
@@ -14,16 +14,22 @@ public static void SignUpSuccess() throws InterruptedException {
         WebDriver driver = new ChromeDriver();
         driver.get("https://deens.com/?internaltraffic");
         driver.manage().window().maximize();
```

Comment:
use @BeforeMethod for preparing your WebDriver and open web-site because this part of code repeat in the begin of each test